docs/APIExamples.rst

```diff
@@ -0,0 +1,570 @@
+=================
+API Examples
+=================
+
+create_version()
+
+Prints a timestamp in DSS_VERSION (ie 1985-04-12T232050.520000Z) that can be used for versioning.    
```

Comment:
it does not print a version it returns it.

Reply:
I find this layout with just a function name as a paragraph hard to read.  Either make it a section header or just incorporate into the text.